src/fsipy/simulations/simulation_common.py

```diff
@@ -0,0 +1,274 @@
+import json
+from typing import List, Union, Tuple
+from pathlib import Path
+
+import numpy as np
+from mpi4py import MPI as mpi
+from dolfin import Mesh, assemble, Constant, MPI, HDF5File, Measure, project, inner, MeshFunction, FunctionSpace, \
+    Function, sqrt, Expression
+
+
+def load_mesh_and_data(mesh_path: str) -> tuple:
```

Comment:
Do we want to do `tuple[dolfin.Mesh, dolfin.MeshFunction, dolfin.MeshFunction]`, or 
```python
from typing import Tuple
Tuple[...]
```
for python < 3.9?

Reply:
I went for the latter.